src/main/java/com/efficio/fieldbook/web/trial/controller/AdvancingController.java

```diff
@@ -594,7 +594,7 @@ protected List<TableHeader> getAdvancedStudyTableHeader() {
 		tableHeaderList.add(new TableHeader(ColumnLabels.DESIGNATION.getTermNameFromOntology(this.ontologyDataManager), "desig"));
 		tableHeaderList.add(new TableHeader(ColumnLabels.PARENTAGE.getTermNameFromOntology(this.ontologyDataManager), "parentage"));
 		tableHeaderList.add(new TableHeader(ColumnLabels.GID.getTermNameFromOntology(this.ontologyDataManager), "gid"));
-		tableHeaderList.add(new TableHeader(ColumnLabels.SEED_SOURCE.getTermNameFromOntology(this.ontologyDataManager), "source"));
+		tableHeaderList.add(new TableHeader("GERMPLASM_ORIGIN", "source"));
```

Comment:
@cuenyad Add a ColumnLabel for GERMPLASM_ORIGIN with null TermId

Reply:
done!.